src/io/flutter/samples/FlutterSampleManager.java

```diff
@@ -37,6 +37,10 @@
     final List<FlutterSample> samples = new ArrayList<>();
     try {
       // TODO(pq): replace w/ index read from repo (https://github.com/flutter/flutter/pull/25515).
+
+      //https://docs.flutter.io/snippets/index.json
```

Comment:
which of these locations is the right one to use?

Reply:
@DanTup : is this now reliably https://master-docs.flutter.io/snippets/index.json?